kolejka/worker/stage0.py

```diff
@@ -122,6 +127,12 @@ def stage0(task_path, result_path, temp_path=None, consume_task_folder=False):
         docker_call += [ '--init' ]
         if task.limits.cpus is not None:
             docker_call += [ '--cpuset-cpus', ','.join([str(c) for c in cgs.limited_cpuset(cgs.full_cpuset(), task.limits.cpus, task.limits.cpus_offset)]) ]
+
+        if task.limits.gpus is not None:
```

Comment:
To check that GPU should be 0 at default (turned off) instead of foreman upperlimit

Reply:
or `> 0`